src/mapml/layers/MapMLLayer.js

```diff
@@ -312,10 +312,18 @@ export var MapMLLayer = LayerGroup.extend({
         onEachFeature: function (properties, geometry) {
           // need to parse as HTML to preserve semantics and styles
           if (properties) {
+            const map = layer._map;
+            const popupOptions = {
+              autoClose: false,
+              autoPan: true,
```

Comment:
I believe the autoPan value was the default `false` and that was intentional so that the map wouldn't keep panning and pulling in new features, which might be a problem for a screenreader user (?).  That was my theory anyhow.  I guess we can keep this value and see if it engenders any comment from anyone else though.